Bottom: 52637b17efade61815ce9a543e6ac319af39e10a
Top:    f577125f6a6e9da1cf989ea2c4e933c66a84cf05
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-10-25 00:19:31 -0700

Refresh of rs-iomap

---

diff --git a/man/rsocket.7 b/man/rsocket.7
index dc3f9ab..bc5bb10 100644
--- a/man/rsocket.7
+++ b/man/rsocket.7
@@ -7,8 +7,8 @@ rsocket \- RDMA socket API
 RDMA socket API and protocol
 .SH "NOTES"
 Rsockets is a protocol over RDMA that supports a socket-level API
-for applications.  rsocket APIs are intended to match the behavior
-of corresponding socket calls, except where noted.  rsocket
+for applications.  Rsocket APIs are intended to match the behavior
+of corresponding socket calls, except where noted.  Rsocket
 functions match the name and function signature of socket calls,
 with the exception that all function calls are prefixed with an 'r'.
 .P
diff --git a/src/rsocket.c b/src/rsocket.c
index 29b8c01..74dbcc7 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -78,7 +78,7 @@ static uint32_t polling_time = 10;
  * bit 29: more data, 0 - end of transfer, 1 - more data available
  *
  * for data transfers:
- * bits [28:0]: bytes transfered
+ * bits [28:0]: bytes transferred
  * for control messages:
  * SGL, CTRL
  * bits [28-0]: receive credits granted
@@ -519,13 +519,6 @@ static int rs_create_ep(struct rsocket *rs)
 	return 0;
 }
 
-/*
-static xxx rs_acquire_iomap_mr(struct rsocket *rs, ...)
-{
-	TODO: write me
-}
-*/
-
 static void rs_release_iomap_mr(struct rs_iomap_mr *iomr)
 {
 	if (atomic_dec(&iomr->refcnt))
